install/ossm-installing-openshift-service-mesh.adoc

```diff
@@ -10,12 +10,18 @@ Installing {ocp-short-name} {SMProductShortName} consists of three main tasks: i
 
 [WARNING]
 ====
-Before migrating from {SMProduct} 2, make sure you are not running {SMProduct} 3 and {SMProduct} 2 in the same cluster, because it causes conflicts unless configured correctly. To migrate from {SMProduct} 2, see xref:../migrating/checklists/ossm-migrating-read-me.adoc#ossm-migrating-read-me[Migrating from {SMProduct} 2.6].
+Before installing {SMProduct} 3, make sure you are not running {SMProduct} 3 and {SMProduct} 2 in the same cluster, because it causes conflicts unless configured correctly. To migrate from {SMProduct} 2, see xref:../migrating/checklists/ossm-migrating-read-me.adoc#ossm-migrating-read-me[Migrating from {SMProduct} 2.6].
 ====
 
 include::modules/ossm-about-deploying-istio-using-service-mesh-operator.adoc[leveloffset=+1]
 include::modules/ossm-about-deployment-and-update-strategies.adoc[leveloffset=+2]
 include::modules/ossm-installing-operator.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="additional-resources-ossm-installing-operator_{context}"]
```

Comment:
The only nit that I have left, and it's annoying that there is a difference, but for additional resource sections like this that go under modules, there is no ID:
```suggestion
```

You can see examples of both types of additional resource sections [here in the guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections)